source/solvers/vof_assemblers.cc

```diff
@@ -77,7 +77,7 @@ VOFAssemblerCore<dim>::assemble_matrix(VOFScratchData<dim>       &scratch_data,
       Tensor<1, dim> velocity_unit_vector =
         velocity / (velocity.norm() + 1e-12);
       const Tensor<2, dim> k_corr =
-        (gradient_unit_vector * velocity_unit_vector) *
+        std::pow(gradient_unit_vector * velocity_unit_vector, 2) *
```

Comment:
Can you use instead https://www.dealii.org/current/doxygen/deal.II/namespaceUtilities.html#a834c6b595ada6f8e73c78d9bbd8ec551


This way this is done at compile time since you know the power at compile time. This is always better :)

Reply:
I'll change all the ones in the file ! :)

Reply:
@blaisb It's done ! There's only the CI left ! :)

Reply:
Thanks for going the extra mile and cleaning things up
:)